src/shell/shell.cpp

```diff
@@ -564,6 +564,8 @@ void SHELL_Init() {
 	MSG_Add("SHELL_CMD_HELP","If you want a list of all supported commands type \033[33;1mhelp /all\033[0m .\nA short list of the most often used commands:\n");
 	MSG_Add("SHELL_CMD_ECHO_ON","ECHO is on.\n");
 	MSG_Add("SHELL_CMD_ECHO_OFF","ECHO is off.\n");
+	MSG_Add("SHELL_ILLEGAL_CONTROL_CHARACTER",
+	        "Unexpected control character: Dec %03u and Hex 0x%02X.\n");
```

Comment:
About: `0x%02X`; Instead of manually adding 0x prefix, printf will format it similarly with `%#04x`.